### DISCUSSION TITLE
Unable to create new projects

### DISCUSSION THREAD

**Discussion by gdcmarinho** [vercel/community#5810] [Help]
### Summary

Hey, buddies!

Everytime I am creating a new project from a template, I got the following error:

An unexpected error occurred. Our team has already been notified and are working to resolve the issue, please try again shortly.

The repository is created, but no files inside.

### Steps to Reproduce

1. Access https://vercel.com/new
2. Choose a template and create
3. The repository in GitHub is created, but no files inside
4. The error message appear: An unexpected error occurred. Our team has already been notified and are working to resolve the issue, please try again shortly.

**Comment by amyegan** [comment]
Hey @gdcmarinho. You may need to adjust the permissions for Vercel under Applications in GitHub settings. I just deployed a new project from the [Next.js Boilerplate](https://vercel.com/templates/next.js/nextjs-boilerplate) template and didn't run into any issues.
